backend/app/routers/garmin_courses.py

```
@@ -39,33 +34,39 @@ def get_db():
 geolocator = geopy.geocoders.Nominatim(user_agent="golfmapper2")
 
 
-@router.get("/readall", status_code=status.HTTP_200_OK)
+from pydantic import BaseModel
```

Ruff (0.14.10)

Module level import not at top of file

(E402)